api/src/main/java/org/openmrs/api/db/hibernate/HibernateConceptDAO.java

```diff
@@ -1027,7 +1028,8 @@ public void remove() {
 	public List<Concept> getConceptsByMapping(String code, String sourceName, boolean includeRetired) {
 		Criteria criteria = createSearchConceptMapCriteria(code, sourceName, includeRetired);
 		criteria.setProjection(Projections.property("concept"));
-		return (List<Concept>) criteria.list();
```

Comment:
Did you intentionally leave out a test for this?

Reply:
Well, I didn't intentionally add a test.  There was no existing test, and the method is deprecated, so I didn't really worry about it.